__tests__/app.test.js

```diff
@@ -380,3 +380,32 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("should return 204 with no content", () => {
+    return request(app)
+      .delete("/api/comments/18")
+      .expect(204)
+      .then(({ body }) => {
```

Comment:
There will never be a body with 204 delete  so we don't need to check the body content here.